dart-runtime/lib/http_client.dart

```diff
@@ -24,6 +24,8 @@ class SdkgenErrorWithData<T> implements Exception {
 
 class SdkgenHttpClient {
   String baseUrl;
+  Map extra;
```

Comment:
We can enforce a specific type for the map keys here

```suggestion
  Map<String, dynamic> extra;
```